Dockerfile

```diff
@@ -27,10 +28,38 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
-ONBUILD ENV PGUSER="$PGUSER" \
+ONBUILD ARG ODOO_MAX_CRON_THREADS=1
+ONBUILD ARG ODOO_WORKERS=0
+ONBUILD ARG ODOO_LIMIT_TIME_REAL=120
+ONBUILD ARG ODOO_LIMIT_TIME_CPU=60
+ONBUILD ARG ODOO_LIMIT_MEMORY_HARD=805306368
+ONBUILD ARG ODOO_LIMIT_MEMORY_SOFT=671088640
+ONBUILD ARG ODOO_LIMIT_REQUEST=8192
+ONBUILD ARG ODOO_LIMIT_TIME_REAL_CRON=1
+ONBUILD ARG ODOO_DB_FILTER=
+ONBUILD ARG ODOO_LIST_DB=1
+ONBUILD ARG ODOO_LOG_LEVEL=info
```

Comment:
Note the above obviously wouldn't work if the configuration is being generated during build, but this also highlights why it should be in the endpoint instead. It seems pretty weird to statically define infrastructure capacity in a Docker image IMO